output_4.2.txt

```diff
@@ -0,0 +1,9 @@
+TASK: 4.2
```

Comment:
+ this file should contain one query - the one that drops index (which is actually missing from 4.1)
+ add query output result